drivers/staging/vc04_services/bcm2835-camera/controls.c

```diff
@@ -1109,7 +1109,7 @@ static const struct bm2835_mmal_v4l2_ctrl v4l2_ctrls[V4L2_CTRL_COUNT] = {
 	{
 		V4L2_CID_POWER_LINE_FREQUENCY, MMAL_CONTROL_TYPE_STD_MENU,
 		0, ARRAY_SIZE(mains_freq_qmenu) - 1,
-		1, 1, NULL,
+		1, 1, mains_freq_qmenu,
```

Comment:
It seems odd to include this patch when the next patch deletes mains_freq_qmenu.

Reply:
The 1st patch got merged upstream, so I brought us into line with upstream before fixing the actual commit.